src/components/Editor/Tabs.css

```diff
@@ -45,12 +45,12 @@
   align-items: center;
   position: relative;
   transition: all 0.15s ease;
-  min-width: 40px;
   overflow: hidden;
   padding: 5px;
   margin-inline-start: 3px;
   margin-top: 3px;
   cursor: default;
+  max-width: 30%;
```

Comment:
Oh, what is this for?

Reply:
Without the 30% max-width, we have the tab go under the toggle button:

<img width="744" alt="nomaxwidth" src="https://user-images.githubusercontent.com/46655/35751442-21d0d014-081e-11e8-8660-1f3baf3c3899.png">